src/toolboxes/spline/UI.tsx

```diff
@@ -1,50 +1,16 @@
-import React, { ChangeEvent, ReactElement } from "react";
-import {
-  Tooltip,
-  IconButton,
-  Accordion,
-  AccordionSummary,
-  Typography,
-  AccordionDetails,
-} from "@material-ui/core";
-
-import { ExpandMore, Timeline, Gesture } from "@material-ui/icons";
+import React, { ReactElement } from "react";
 
 import { Tool } from "@/tools";
 
 interface Props {
-  expanded: boolean;
+  open: boolean;
   activeTool: Tool;
-  onChange: (event: ChangeEvent, isExpanded: boolean) => void;
+  anchorEl: any;
+  onClose: (event: React.MouseEvent) => void;
+  onClick: (event: React.MouseEvent) => void;
   activateTool: (tool: Tool) => void;
 }
 
-const SplineUI = (props: Props): ReactElement => (
-  <Accordion expanded={props.expanded} onChange={props.onChange}>
-    <AccordionSummary expandIcon={<ExpandMore />} id="spline-toolbox">
-      <Typography>Splines</Typography>
-    </AccordionSummary>
-    <AccordionDetails>
-      <Tooltip title="Activate spline">
-        <IconButton
-          id="activate-spline"
-          onClick={() => props.activateTool("spline")}
-          color={props.activeTool === "spline" ? "secondary" : "default"}
-        >
-          <Timeline />
-        </IconButton>
-      </Tooltip>
-      <Tooltip title="Magic spline">
-        <IconButton
-          id="activate-magic-spline"
-          onClick={() => props.activateTool("magicspline")}
-          color={props.activeTool === "magicspline" ? "secondary" : "default"}
-        >
-          <Gesture />
-        </IconButton>
-      </Tooltip>
-    </AccordionDetails>
-  </Accordion>
-);
+const SplineUI = (props: Props): ReactElement => null;
```

Comment:
Given that you have taken all the content out of this file, is there a reason you haven't just deleted it?

Reply:
This does seem to have happened... personally, I am slightly concerned that it has happened... maybe one for discussion at tomorrow's daily?

Reply:
Yea, I'll have a closer look at this and see if it makes sense to pick one way over the over

Reply:
Created an issue for this to revisit later #195